### DISCUSSION TITLE
Deleting specific routing point

### DISCUSSION THREAD

**Discussion by EightM** [eclipse-glsp/glsp#1288] [Q&A] [answered]
Hello!
Is it possible to delete a specific routing point from an edge? Delete operation deletes the edge entirely. The only way I've found is to straighten the bend. 

I want operation like this (gif):
![chrome_df2YaeXJa2](https://github.com/eclipse-glsp/glsp/assets/4703545/b734737d-cda2-47e8-a3cb-2bf1637c3b07)

**Comment by tortmayr** [comment, marked as answer]
Hi @EightM 

unfortunatley this is currently not supported. As you already noticed, the only way to `delete` a routing point is to straighten the bend.
However, this is a very reasonable request, please feel free to open a feature request for this in https://github.com/eclipse-glsp/glsp/issues
It should be doable by adapting the `DelKeyDeleteTool` and the `DeleteMouseTool`. If the element to delete is a routing handle we have to retrieve the corresponding routing point (index) and send a `ChangeRoutingPointsOperation` with the adapted points to the server instead of a `DeleteElementOperation`.
